pandas/tests/io/test_parquet.py

```diff
@@ -812,11 +812,15 @@ def test_write_with_schema(self, pa):
     def test_additional_extension_arrays(self, pa):
         # test additional ExtensionArrays that are supported through the
         # __arrow_array__ protocol
+
+        from pandas.core.arrays.string_arrow import ArrowStringDtype  # noqa: F401
```

Comment:
force registration of "arrow_string", not strictly necessary but won't be missed as `ArrowStringDtype` will be removed.